feed-template.php

```diff
@@ -1,6 +1,4 @@
 <?php
-header('Content-Type: application/json; charset=' . get_option('blog_charset'));
```

Comment:
WordPress already adds this here https://github.com/WordPress/WordPress/blob/493f76a3d2ef8c030ab5dcd4333f9a401208f534/wp-includes/class-wp.php#L423